src/playback/config.js

```diff
@@ -124,6 +124,7 @@ const PLATFORMS = [
 const BROWSERS =[
   {id: "firefox", label:"Firefox", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-firefox-live"]}}},
   {id: "geckoview", label:"geckoview", filter: {eq: {framework: 10, repo: 'mozilla-central', suite: ["raptor-youtube-playback-geckoview-live"]}}},
+  {id: "fenix", label:"fenix", filter: {eq: {framework: 10, repo: 'fenix', suite: ["raptor-youtube-playback-fenix-live"]}}},
```

Comment:
This should be labeled as "Firefox Preview". Also note in the deploy preview that there are several graphs with no data or data older than 3 days. I think we should increase the no data warning to a week on fenix.

Reply:
@davehunt So should we increase the missing data interval to 7 only on fenix and let firefox and geckoview have the default of 3?

Reply:
Yes, we expect Firefox and GeckoView to have multiple results a day. Fenix runs once/day but there may be no new commits for several days, so the results will appear to be older.